### PR TITLE
Adrianz is fixing stuff

### PR DESCRIPTION
**Pull request by Adrianzatreanu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,8 +51,10 @@ Docs
 __Team Leader:__ Adrian Zatreanu
 (adrianzatreanu@coala.io)
 
-Improve the existing documentation, wiki, tutorials and guides.
-Introduce new ones as needed.
+The documentation team aims having updated docs which maintain consistency and
```
Comment (sims1253) [inline README.md]: This sentence doesn't work

File: README.md [hunk 2]
```diff
@@ -48,8 +51,10 @@ Docs
...
+reduce redundancy of having the same thing across different pages.
+It also takes care of wiki pages, events, and everything that is related
+to the non-coding side of coala.
```
Comment (sims1253) [inline README.md]: Ok, let me just hand over all the things I have on my community team list :D

**Comment by sims1253** [comment]
ack 99d8adc
ack 8da6176

File: README.md (hunk 1 above)
Comment (Adrianzatreanu) [reply README.md]: how would you rephrase it?

File: README.md (hunk 2 above)
Comment (Adrianzatreanu) [reply README.md]: hehe, maybe i went into the wrong direction :D

**Comment by sims1253** [comment]
Please take a look at [the community team idea list](https://github.com/coala/teams/blob/master/community/community_ideas.md)
We might want to work together in some areas and draw clean lines in other so we don't overlap

**Comment by Adrianzatreanu** [comment]
reack 24c471f 71c2b32

**Review by sils — APPROVED** [review]

**Comment by sils** [comment]
@rultor merge

**Comment by sils** [comment]
oh needs rebase I guess

**Comment by sils** [comment]
@Adrianzatreanu can you rebase and merge this?

**Comment by Adrianzatreanu** [comment]
reack a82ba23 bca4c51 04554da

**Comment by Adrianzatreanu** [comment]
@rultor merge

**Pull request merged**